bin/spark-sql

```diff
@@ -43,15 +43,12 @@ function usage {
   echo
   echo "CLI options:"
   "$FWDIR"/bin/spark-class $CLASS --help 2>&1 | grep -v "$pattern" 1>&2
+  exit $1
```

Comment:
I don't think this is working correctly, I'm just getting two instances of the `spark-submit` style help when I run:

```
./bin/spark-sql --help
Usage: ./bin/spark-sql [options] [cli option]
```